app/main.py

```diff
@@ -1,6 +1,23 @@
+import functools
 from typing import Callable
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached_data = {}
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs) -> any:
+        kwargs_value = tuple()
+        for key, value in kwargs.items():
+            kwargs_value += (key, value)
```

Comment:
You could convert `kwargs` to string and store them in this way avoiding iteration